documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -112,26 +194,98 @@ docker run -it --rm --name connect \
     -e CONNECT_VALUE_CONVERTER_APICURIO_REGISTRY_GLOBAL-ID=io.apicurio.registry.utils.serde.strategy.AutoRegisterIdStrategy \
     -p 8083:8083 debezium/connect-apicurio:{debezium-docker-label}
 ----
+endif::community[]
+ifdef::product[]
+
+.. Follow the steps in the link:{LinkCDCGettingStarted}#deploying-kafka-connect[example of deploying Kafka Connect] in {NameCDCGettingStarted}.
```

Comment:
@TovaCohen, I somehow had missed this before (thanks to @jcechace for pointing it out), here you're adding new S2I docs. While we agreed to only fully transition to the custom containers approach for the next release docs, why adding more S2I-flavoured this time, which we only will have to modify next time?

Reply:
@gunnarmorling I agree that this is the wrong way to update the doc. But I don't know how to make it right. Please see my previous comment.  I recognize the inconsistency and the disconnect in the procedure. But I don't have any background with regard to OpenShift images. I plan to remedy that. I'll go do some reading now. But I need precise input from you or Jiri or someone to correct this documentation for this release.

Reply:
Not only that... this procedure starts with Dockerfile (correct way of building custom Kafka Connect image) and then switches to s2i... this creates a mix between two working (although we want to promote only the Docker way) approaches...